server/common.js

```diff
@@ -33,7 +37,9 @@ const query = {
 	INSERT_ACCOUNT: "INSERT INTO user (name, email, password) VALUES (?,?,?)",
 	INSERT_TWEET: "INSERT INTO tweets (email, tweet, timestamp) VALUES (?,?,?)",
 	CREATE_USER_TABLE: "CREATE TABLE IF NOT EXISTS user (email text PRIMARY KEY UNIQUE, password text, name text, CONSTRAINT email_unique UNIQUE (email))",
-	CREATE_TWEETS_TABLE: "CREATE TABLE IF NOT EXISTS tweets (email text, tweet text, timestamp integer)"
+	CREATE_TWEETS_TABLE: "CREATE TABLE IF NOT EXISTS tweets (email text, tweet text, timestamp integer)",
+
+	UPDATE_PASSWORD: "UPDATE user SET password = ? WHERE email = ? AND name = ?"
```

Comment:
Do we care about name here? I think `"UPDATE user SET password = ? WHERE email = ?" would work fine -- this way you don't need to pass name in the requests